test/e2e/networkpolicy_test.go

```diff
@@ -440,6 +440,13 @@ func createAndWaitForPod(t *testing.T, data *TestData, createFunc func(name stri
 func waitForAgentCondition(t *testing.T, data *TestData, podName string, conditionType v1beta1.AgentConditionType, expectedStatus corev1.ConditionStatus) {
 	if err := wait.Poll(1*time.Second, defaultTimeout, func() (bool, error) {
 		cmds := []string{"antctl", "get", "agentinfo", "-o", "json"}
+		if testOptions.enableCoverage {
+			const timeFormat = "20060102T150405Z0700"
+			timeStamp := time.Now().Format(timeFormat)
+			cmds = []string{"antctl-coverage", "-test.run=TestBincoverRunMain", fmt.Sprintf("-test.coverprofile=antctl-%s.out", timeStamp), "get", "agentinfo", "-o", "json"}
+		}
```

Comment:
I would suggest sticking to the `antctl` binary here (not `antctl-coverage`) for the sake of simplicity. These invocations are not meant to test any `antctl` code, and if we had a better way to check for the condition than using `antctl`, then we would not be using it at all IMO

Reply:
this also make sense. Then we can only change test/e2e/antctl_test.go file for test of antctl code.

Reply:
same applies for traceflow_test.go and security_test.go

Reply:
thanks, removed the changes in traceflow_test.go and security_test.go and basic_test.go